docs/connecting-your-data/data-warehouses/connecting-to-redshift.md

```diff
@@ -1,6 +1,6 @@
 # Connecting to Redshift
 
-## 1. Whitelist Eppo IP Address
+## 1. Allowlist Eppo IP Address
```

Comment:
Nit: I like this change but allowlist sounds more like a noun than a verb to me, making this sound a little odd, maybe just `Allow Eppo IP address`?

Reply:
Could also be `Add Eppo IP addresses to Allowlist`